networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/rpc/methods/GoodbyeMessageHandler.java

```diff
@@ -39,7 +39,7 @@ public GoodbyeMessageHandler(final MetricsSystem metricsSystem) {
   }
 
   @Override
-  public Void onIncomingMessage(final Peer peer, final GoodbyeMessage message) {
+  public GoodbyeMessage onIncomingMessage(final Peer peer, final GoodbyeMessage message) {
```

Comment:
We can't return `Void` anymore because it doesn't implement `SimpleOffsetSerializable`.  The req/resp framework doesn't currently support not sending a response (Goodbye will wind up throwing a `NullPointerException`) so we'll sort out what the right thing to do here is as part of properly supporting `GOODBYE` messages.